ui/main/src/app/business/services/realtime-domain.service.ts

```
@@ -109,10 +109,15 @@
     /**
      * apply new timeline domain
      * feed state dispatch a change on filter, provide the new filter start and end
-     * @param startDomain new start of domain
-     * @param endDomain new end of domain
+     * @param startPeriod new start of domain
+     * @param endPeriod new end of domain
      */
-    public static setStartAndEndDomain(startDomain: number, endDomain: number, useOverlap = false) {
+    public static setStartAndEndPeriod(startPeriod: number, endPeriod: number) {
+        this.currentDomainId = 'Unsynchronised';
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)